packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js

```diff
@@ -404,7 +407,29 @@ export default class AgoraIpV4OrderController {
 
     const ipOrganisationPromise = this.IpOrganisation.getIpOrganisation().then(
       (organisations) => {
-        this.organisations = organisations;
+        if (
+          this.model.selectedService?.type ===
+          PRODUCT_TYPES.dedicatedServer.typeName
+        ) {
+          let registry = null;
+
+          switch (REGION) {
+            case SERVER_REGION.EUROP:
+              registry = ORGANISATION_GROUP.RIPE;
+              break;
+            case SERVER_REGION.USA:
+            case SERVER_REGION.CANADA:
+              registry = ORGANISATION_GROUP.ARIN;
+              break;
+            default:
+              registry = null;
+          }
+          this.organisations = registry
+            ? organisations.filter((org) => org.registry === registry)
+            : organisations;
+        } else {
```

Comment:
I guess we should't have a else case. This change is for all the service not only for dedicated server. We need to retain this for all service type.